submissions/unabyband/oop_exercise/index.js

```diff
@@ -0,0 +1,87 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/unabyband/a-tiny-JS-world
+   Web app: https://unabyband.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor (species, name, gender, limbs, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.limbs = limbs;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  printInhabitant () {
+    print(`${this.species}; <strong>${this.name}</strong>; ${this.gender}; ${this.limbs.join('; ')}; <em>${this.saying}</em>; ${this.friends ? this.friends.map(elem => elem.name).join(', ') : 'Cats have no friends'}`);
```

Comment:
There are `Array` methods to make code DRYer. An array element can be an expression (or rather a result of expression).

Reply:
Done!

Reply:
`Object` methods still do not guarantee order of properties/values.
You could improve exisiting line 22.

Reply:
Do I need deconstruct this method?

Reply:
Not really. We speak about `[].join` and
> An array element can be an expression (or rather a result of expression).

Feel free to ask questions

Reply:
But our inhabitants are objects, not the arrays, do we need to convert them or use some sort of map method?
I'm not sure if this will make my code DRYer...

Reply:
Ok, I think I finally get it, please review again

Reply:
Could you review it again, please?